### PR TITLE
[BIP-373] Slight rewrite of evenness byte footnote for clarity

### PR DESCRIPTION
**Pull request by kdmukai**
Per discussion with @jonatack here: https://github.com/bitcoin/bips/pull/1705#discussion_r1879264639

### CODE REVIEW

File: bip-0373.mediawiki [hunk 1]
```diff
@@ -52,9 +52,11 @@ The new per-input types are defined as follows:
...
 |-
 | The MuSig2 aggregate public key (compressed) <ref>'''Why the compressed aggregate public key instead of x-only?'''
-BIP 32 requires public keys to include their evenness byte. Aggregate public keys are expected to be
-derived from, following [[bip-0328.mediawiki|BIP 328]], and therefore will
-need to include the evenness. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
+BIP 32 public keys can be derived from a BIP 327 MuSig2 aggregate public key (see: [[bip-0328.mediawiki|BIP 328]]).
+But since BIP 32 requires public keys to include their evenness byte, BIP 327 MuSig2 aggregate public keys must
+include their evenness byte as well.
+
+Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
```
Comment (jonatack) [inline bip-0373.mediawiki:59]: Not sure about adding the inoperative newlines within this footnote, feel free to ignore.

**Review by jonatack — COMMENTED** [review]
ACK be81c8a1a11e083475f27ed1db77fcec969bd01e

File: bip-0373.mediawiki (hunk 1 above)
Comment (kdmukai) [reply bip-0373.mediawiki:59]: Wasn't sure which was preferred: making space to preserve the unedited text (easiest diff to review) or reformatting/reflowing the entire paragraph with new line breaks (nicest final form).

Comment (achow101) [reply bip-0373.mediawiki:59]: The extra blank line does not render as a paragraph break, if that was what was intended.

File: bip-0373.mediawiki [hunk 2]
```diff
@@ -52,9 +52,11 @@ The new per-input types are defined as follows:
...
 to identify master keys, and these fingerprints require y-coordinate of the public key,
```
Comment (jonatack) [inline bip-0373.mediawiki:60]: While here can do this as well https://github.com/bitcoin/bips/pull/1705#discussion_r1874516388

```suggestion
to identify master keys, and these fingerprints require the y-coordinate of the public key,
```

File: bip-0373.mediawiki (hunk 1 above)
Comment (jonatack) [reply bip-0373.mediawiki:59]: Yes, probably clearer not to add them.

Comment (kdmukai) [reply bip-0373.mediawiki]: * Updated to remove invisible paragraph break.
* Raw text reflowed to fit within 120 col width to be more tidy (even though has no impact on how it's actually rendered).

File: bip-0373.mediawiki (hunk 2 above)
Comment (kdmukai) [reply bip-0373.mediawiki]: Missing "the" added.

File: bip-0373.mediawiki [hunk 3]
```diff
@@ -52,17 +52,16 @@ The new per-input types are defined as follows:
...
-partial signature.</ref> from the <tt>KeyAgg</tt> algorithm. This key may or may not appear
-(as x-only) in the Taproot output key, the internal key, or in a script. It may instead be a parent public
-key from which the Taproot output key, internal key, or keys in a script were derived.
+BIP 32 public keys can be derived from a BIP 327 MuSig2 aggregate public key (see: [[bip-0328.mediawiki|BIP 328]]).
+But since BIP 32 requires public keys to include their evenness byte, BIP 327 MuSig2 aggregate public keys must
+include their evenness byte as well. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints to identify
+master keys, and these fingerprints require the y-coordinate of the public key, so x-only serialization can't be used.
+By including the aggregate key as a full public key, signers that are unaware of the MuSig2 outside of the PSBT will
+still be able to identify which keys are derived from the aggregate key by computing and then comparing the
+fingerprints. This is necessary for the signer to apply the correct tweaks to their partial signature.</ref> from the
```
Comment (jonatack) [inline bip-0373.mediawiki:61]: @kdmukai while here, do you think "This" is unambiguous enough, or could it be clearer?

**Review by jonatack — COMMENTED** [review]
ACK overall change 45e626feab08d80093f622c9c2441bc127cf14c0 and review of `git diff be81c8a 45e626f`

File: bip-0373.mediawiki (hunk 3 above)
Comment (kdmukai) [reply bip-0373.mediawiki:61]: I think it's clear enough, though that detail is getting too far beyond my expertise so it's inherently word-soupy to me (i.e. it's a "me" problem, not a text problem).

**Comment by achow101** [comment]
Post merge ACK

If you're going to request my review, please at least let me review it before merging.

**Pull request merged**